### DISCUSSION TITLE
Livewire sometimes won't work inside x-data alpine component

### DISCUSSION THREAD

**Discussion by gbekura** [alpinejs/alpine#4083] [7. Bugs (archived)] [answered]
Suddenly my Livewire app stopped responding to wire:click or wire:model related events, livewire components were showing normally, but it would not fire any events. And after hours of pain and tests i discovered that you should not include alpine cnd if you have latest livewire, this solved problem.

**Comment by joshhanley** [comment, marked as answer]
@gbekura that's not a bug as Livewire V3 now comes bundled with Alpine, it's in the installation manual 🙂 https://livewire.laravel.com/docs/quickstart#install-livewire

<img width="802" alt="image" src="https://github.com/alpinejs/alpine/assets/882837/5a671ba9-6877-4e4d-ab1a-9de2fab1407b">

**Reply by gbekura** [reply]
I feel so ashamed. Thank you very much! ❤️ And thank you for making development easier ❤️
